zero/src/prover.rs

```diff
@@ -10,8 +10,10 @@ use alloy::primitives::U256;
 use anyhow::{Context, Result};
 use evm_arithmetization::Field;
 use evm_arithmetization::SegmentDataIterator;
+use futures::future::try_join_all;
 use futures::{
-    future, future::BoxFuture, stream::FuturesUnordered, FutureExt, TryFutureExt, TryStreamExt,
+    future, future::BoxFuture, stream::FuturesUnordered, FutureExt, StreamExt, TryFutureExt,
+    TryStreamExt,
```

Comment:
nit: merge imports, extension traits should be anonymous
```suggestion
    future:{self, try_join_all}, future::BoxFuture, stream::FuturesUnordered, FutureExt as _, StreamExt as _, TryFutureExt as _,
    TryStreamExt as _,
```